### PR TITLE
feat: support workflows and extra args to terraform executions

### PR DESCRIPTION
**Pull request by Spartakovic**

### CODE REVIEW

File: pkg/digger/digger_config.go [hunk 1]
```diff
@@ -18,15 +17,30 @@ type WorkflowConfiguration struct {
...
+	Steps []Step `yaml:"steps"`
+}
+
+type Workflow struct {
+	Plan          *Stage                 `yaml:"plan,omitempty"`
+	Apply         *Stage                 `yaml:"apply,omitempty"`
+	Configuration *WorkflowConfiguration `yaml:"workflow_configuration"`
+}
+
+type Command struct {
```
Comment (motatoes) [inline pkg/digger/digger_config.go]: this isn't used anywhere, its part of a step now, should it be removed?

File: pkg/digger/digger_config.go [hunk 2]
```diff
@@ -50,6 +56,28 @@ func (p *Project) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 }
 
+type Step struct {
+	Action    string
```
Comment (motatoes) [inline pkg/digger/digger_config.go:60]: this needs `yaml:"action"` appended

Comment (Spartakovic) [reply pkg/digger/digger_config.go:60]: no, it's not 1-2-1 mapping between yaml and go struct

**Review by veziak — APPROVED** [review]
lgtm

**Pull request merged**